plugins/redis-cache/src/index.ts

```diff
@@ -0,0 +1,157 @@
+import type {
+  AmplicationPlugin,
+  CreateServerAppModuleParams,
+  CreateServerDockerComposeParams,
+  CreateServerDotEnvParams,
+  CreateServerPackageJsonParams,
+  DsgContext,
+  Events
+} from "@amplication/code-gen-types";
+import { EventNames } from "@amplication/code-gen-types";
+import { merge } from "lodash"
+import * as utils from "./utils"
+import { builders, namedTypes } from "ast-types"
+import * as constants from "./constants"
+
+
+
+class RedisCachePlugin implements AmplicationPlugin {
+  register(): Events {
+    return {
+      [EventNames.CreateServerPackageJson]: {
+        before: this.beforeCreateServerPackageJson
+      },
+      [EventNames.CreateServerAppModule]: {
+        before: this.beforeCreateServerAppModule
+      },
+      [EventNames.CreateServerDotEnv]: {
+        before: this.beforeCreateServerDotEnv
+      },
+      [EventNames.CreateServerDockerCompose]: {
+        before: this.beforeCreateServerDockerCompose
```

Comment:
This looks good. I'd suggest to also create an event handler for the event `CreateServerDockerComposeDev`.
This event will handle the creation of `docker-compose.dev.yml` a composition with all services needed by `server` to allow running `server` without docker.

Reply:
Hey @overbit, I've made the changes.